src/lib/util.js

```diff
@@ -887,3 +875,12 @@ export function getProp(object, propPath, defaultValue) {
 
     return value !== undefined ? value : defaultValue;
 }
+
+/**
+ * Returns whether Preview is running inside the Box web application.
+ *
+ * @return {boolean} Is Preview running in the Box WebApp
+ */
+export function inBoxWebApp() {
+    return (window.location.hostname || '').indexOf('app.box.com') !== -1;
```

Comment:
We have an 'includes' polyfill if you want to use that
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes

Reply:
Actually I'm gonna hold off adding a polyfill for now. The existing polyfill is for Array.prototype.includes. If I included the String polyfill, I'd want to update all references of `indexOf` and that looks like quite a few files.

Reply:
Seems like we follow an isSomething format for boolean returning functions. Maybe isBoxWebApp, or the inverse function isPlatformApp